script/ModuleRegistry.sol

```diff
@@ -0,0 +1,400 @@
+            string memory name = abi.decode(constructorArgs, (string));
+            return address(new TransactionForwarder_v1(name));
+        }
+        // Factories
+        else if (Strings.equal(contractName, "ModuleFactory_v1")) {
+            (address reverter, address forwarder) =
+                abi.decode(constructorArgs, (address, address));
+            return address(new ModuleFactory_v1(reverter, forwarder));
+        } else if (Strings.equal(contractName, "OrchestratorFactory_v1")) {
+            address forwarder = abi.decode(constructorArgs, (address));
+            return address(new OrchestratorFactory_v1(forwarder));
+        }
+        // Funding Managers
+        else if (Strings.equal(contractName, "FM_Rebasing_v1")) {
+            return address(new FM_Rebasing_v1());
+        } else if (
+            Strings.equal(
+                contractName, "FM_BC_Bancor_Redeeming_VirtualSupply_v1"
+            )
+        ) {
+            return address(new FM_BC_Bancor_Redeeming_VirtualSupply_v1());
+        } else if (
+            Strings.equal(
+                contractName,
+                "FM_BC_Restricted_Bancor_Redeeming_VirtualSupply_v1"
+            )
+        ) {
+            return address(
+                new FM_BC_Restricted_Bancor_Redeeming_VirtualSupply_v1()
+            );
+        }
+        // Authorizer
+        else if (Strings.equal(contractName, "AUT_Roles_v1")) {
+            return address(new AUT_Roles_v1());
+        } else if (Strings.equal(contractName, "AUT_TokenGated_Roles_v1")) {
+            return address(new AUT_TokenGated_Roles_v1());
+        } else if (Strings.equal(contractName, "AUT_EXT_VotingRoles_v1")) {
+            return address(new AUT_EXT_VotingRoles_v1());
+        }
+        // Payment Processors
+        else if (Strings.equal(contractName, "PP_Simple_v1")) {
+            return address(new PP_Simple_v1());
+        } else if (Strings.equal(contractName, "PP_Streaming_v1")) {
+            return address(new PP_Streaming_v1());
+        }
+        // Logic Modules
+        else if (Strings.equal(contractName, "LM_PC_PaymentRouter_v1")) {
+            return address(new LM_PC_PaymentRouter_v1());
+        } else if (Strings.equal(contractName, "LM_PC_Bounties_v1")) {
+            return address(new LM_PC_Bounties_v1());
+        } else if (Strings.equal(contractName, "LM_PC_RecurringPayments_v1")) {
+            return address(new LM_PC_RecurringPayments_v1());
+        } else if (Strings.equal(contractName, "LM_PC_KPIRewarder_v1")) {
+            return address(new LM_PC_KPIRewarder_v1());
+        } else if (Strings.equal(contractName, "LM_PC_Staking_v1")) {
+            return address(new LM_PC_Staking_v1());
+        }
+        revert("Unknown Module");
+    }
+
+    //  FACTORIES AND BEACONS
+    //--------------------------------------------------------------------------
+
+    // OrchestratorFactory
+    address orchestratorFactory;
```

Comment:
You dont save the implementation addresses here.
Did you want to use them later or am I missing something?

Reply:
typo, fixed it